src/main/scala/org/apache/spark/ml/ModelBuilder.scala

```diff
@@ -60,7 +60,7 @@ object ModelBuilder {
   def newLinearSVCModel(
       coefficients: Vector,
       intercept: Double): LinearSVCModel = {
-    new LinearSVCModel("linearSVC", coefficients, intercept)
+    ClassificationModelBuilder.newLinearSVCModel(coefficients, intercept)
```

Comment:
The `LinearSVC` has a modifier private[classification] so I have to move it to there.